onnxruntime/core/providers/acl/tensor/concat.cc

```diff
@@ -32,7 +32,7 @@ Status Concat<T>::Compute(OpKernelContext* ctx) const {
     input_tensors.push_back(ctx->Input<Tensor>(i));
   }
 
-  std::vector<int64_t> output_dims = input_tensors[0]->Shape().GetDims();
+  std::vector<int64_t> output_dims = input_tensors[0]->Shape().GetDimsAsVector();
```

Comment:
>GetDimsAsVector [](http://example.com/codeflow?start=63&length=15)

This function no longer exists. See coding standards. All shapes are now handled with TensorShapeVector which has small buffer optimization

Reply:
Well, I was using v1.10.0 and testing in the version of code, in which `GetDimsAsVector` exists but `AsShapeVector` does not. I will change to `AsShapeVector` as suggested.

Reply:
The function does not return std::vector. Please, read the comment above.

Reply:
Just replace `std::vector<int64_t>` with `auto`, should work.